frontend/lib/rental-history.tsx

```diff
@@ -0,0 +1,204 @@
+        <FormConfirmAddressModal toStep2={Routes.locale.rh.preview} />
+      )} />
+    </Page>
+  );
+}
+
+function RentalHistoryPreview(): JSX.Element {
+  const appContext = useContext(AppContext);
+  const formData = appContext.session.rentalHistoryInfo;
+
+  return (
+    <Page title="Review your email to the DHCR">
+      <h1 className="title is-4">Review your request to the DHCR</h1>
+      <p>Here is a preview of the request for your rent history. It includes your address and apartment number so that the DHCR can mail you.</p>
+        <br />
+      {formData &&
+        <article className="message">
+          <div className="message-header">
+            <p className="has-text-weight-normal">To: New York Division of Housing and Community Renewal (DHCR)</p>
+          </div>
+          <div className="message-body">
+            <h4 className="is-italic">Subject: {rhEmailText.DHCR_EMAIL_SUBJECT}</h4>
+              <div className="is-divider jf-divider-narrow" />
+            <p>DHCR administrator,</p>
+              <br />
+            <p>
+              {rhEmailText.DHCR_EMAIL_BODY
+                .replace('FULL_NAME', formData.firstName + ' ' + formData.lastName)
+                .replace('FULL_ADDRESS', formData.address + ', ' + getBoroughChoiceLabels()[formData.borough as BoroughChoice])
+                .replace('APARTMENT_NUMBER', formData.apartmentNumber)}
+            </p>
+              <br />
+            <p>{rhEmailText.DHCR_EMAIL_SIGNATURE} </p>
+            <p>{formData.firstName + ' ' + formData.lastName}</p>
+          </div>
+        </article>
+      }
+      <div className="field is-grouped jf-two-buttons">
+        <BackButton label="Back" to={Routes.locale.rh.form} />
+        <SessionUpdatingFormSubmitter
+          mutation={RhSendEmailMutation}
+          initialState={{}}
+          onSuccessRedirect={Routes.locale.rh.confirmation}
+        >
+          {(ctx) => 
+          <NextButton label="Submit request" isLoading={ctx.isLoading} /> }
+        </SessionUpdatingFormSubmitter>
+      </div> 
+    </Page>
+  );
+}
+
+function RentalHistoryConfirmation(): JSX.Element {
+  return (
+    <Page title="Your rent history has been requested!" withHeading="big" className="content">
+      <h2>What happens next?</h2>
+      <p>You should receive your rent history in the mail in about a week. If you have more questions, please email us at <a href="mailto:support@justfix.nyc" target="_blank" rel="noopener noreferrer">support@justfix.nyc</a>.</p>
+      <Link to={Routes.locale.dataDrivenOnboarding} className="button is-primary is-medium">Explore our other tools</Link>
+      <h2>Want to read more about your rights?</h2>
+      <ul>
+        <li><a href="http://metcouncilonhousing.org/help_and_answers" target="_blank">MetCouncil on Housing</a></li>
+        <li><a href="http://housingcourtanswers.org/glossary/" target="_blank">Housing Court Answers</a></li>
```

Comment:
You should use an [`<OutboundLink>`](https://github.com/JustFixNYC/tenants2/blob/c958ce12a2f637d3c45bdfe983bdaab2b87cb7f2/frontend/lib/pages/loc-confirmation.tsx#L57) instead here to ensure that Google Analytics will track the outbound link click.

Reply:
this was covered in #887!